### DISCUSSION TITLE
Browser Native Back Forward buttons keeping scrolling on top and there is no way to disable it

### DISCUSSION THREAD

**Discussion by bayraak** [vercel/next.js#22573] [Help]
Adding `scroll: false` to `<Link />` is okay but there is no way to globally, application-level to disable scrolling after clicking to any link.

In my use-case since I'm opening a modal and routing to an URL, when I need to go back to close the modal it scrolls the page up and loses track.

Is there a way how we can disable scrolling also for browser back/forward buttons?

**Comment by greynguyen** [comment]
I have a similar question

**Comment by bayraak** [comment]
Any help or proposed solution?
